### PR TITLE
Add pyup config

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: .pyup.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+schedule: "every day"
+search: False
+update: all
+requirements:
+  - requirements/default.txt:
+      update: all
+  - requirements/constraints.txt:
+      update: all
```
[the remark this answers is not in the document]
Comment (leplatrem) [reply .pyup.yml]: I think I always want to have the latest versions on master :)

[the remark this answers is not in the document]
Comment (leplatrem) [reply .pyup.yml]: I think I always want to have the latest versions on master :)

**Comment by leplatrem** [comment]
@peterbe I was suggested to enable dependabot.

Also because it supports Pipenv, which I would prefer over the default/constraints approach that we have here

**Comment by peterbe** [comment]
So is dependabot better than PyUP? Should we switch to that for all other projects? 

I can't remember why we still bother with requirements.txt files. Is it because of pyup required it or is it because of Docker?

**Comment by leplatrem** [comment]
> So is dependabot better than PyUP? Should we switch to that for all other projects?

I was suggested it when I asked to enable Pyup in this repo. 

I can't really say if we should move every repo... Let's give it try it and decide ;) 

> I can't remember why we still bother with requirements.txt files. Is it because of pyup required it or is it because of Docker?

Before we had setup.py + pinned versions in requirements.txt 

But yes, this project is meant to be deployed via Docker.  I would say that Pipenv would do a better job, but that's not on my top-priority list ;)

**Pull request merged**